standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-4.0.0-alpha-2-to-4.0.0.mysql.sql

```diff
@@ -1,5 +1,9 @@
 SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-2 to 4.0.0' AS MESSAGE;
 
+-- HIVE-26221
+ALTER TABLE TAB_COL_STATS ADD HISTOGRAM blob;
+ALTER TABLE PART_COL_STATS ADD HISTOGRAM blob;
```

Comment:
This query will add `HISTOGRAM` column to the tail of table `PART_COL_STATS`, while in https://github.com/apache/hive/pull/3137/files#diff-7f6669046b6201c6e173b680e06aad8dda6ecfcd4624e990c43e4256fff33c68, the column sits before the column `AVG_COL_LEN`, is it intended?